search-service/src/main/kotlin/com/egm/stellio/search/service/TemporalEntityAttributeService.kt

```diff
@@ -615,17 +615,21 @@ class TemporalEntityAttributeService(
                     )
                 }.bind()
             } else {
-                val message = if (ngsiLdAttributeInstance.datasetId != null)
-                    "Attribute (datasetId: ${ngsiLdAttributeInstance.datasetId}) does not exist"
-                else
-                    "Attribute (default instance) does not exist"
-                logger.info(message)
-                UpdateAttributeResult(
+                addAttribute(
+                    entityUri,
                     ngsiLdAttribute.name,
```

Comment:
is this to treate the case where a temporal entity doesn't have any instances of this attribute yet ?

Reply:
in 1.6, new behavior states that if the attribute does not exist, it has to be added (before it had to be ignored)